### PR TITLE
Story/update readme for 1 1 0

### PR DESCRIPTION
**Pull request by igorgonibm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,23 @@
...
-  - IBM FlashSystem A9000/R
-
+  - IBM DS8880
+  - IBM DS8900
+  
 Supported operating systems:
-  - RHEL 7.x (x86 architecture)
-
+  - Red Hat Enterprise Linux 7.x (x86 architecture)
+  - Red Hat Enterprise Linux CoreOS (x86, IBM Z architectureS)
```
Comment (oriyarde) [inline README.md]: `architechtureS`

File: README.md [hunk 2]
```diff
@@ -148,21 +134,22 @@ metadata:
...
-  csi.storage.k8s.io/controller-publish-secret-name: <VALUE_ARRAY_SECRET>
-  csi.storage.k8s.io/controller-publish-secret-namespace: <VALUE_ARRAY_SECRET_NAMESPACE>
+  csi.storage.k8s.io/provisioner-secret-name: svc-array
+  csi.storage.k8s.io/provisioner-secret-namespace: csi-ns
+  csi.storage.k8s.io/controller-publish-secret-name: svc-array
+  csi.storage.k8s.io/controller-publish-secret-namespace: csi-ns
 
-  csi.storage.k8s.io/fstype: xfs   # Optional: Values ext4/xfs. The default is ext4.
+  csi.storage.k8s.io/fstype: xfs   # Optional. values ext4\xfs. The default is ext4.
+  volume_name_prefix: demo         # Optional.
```
Comment (oriyarde) [inline README.md]: I thought we want to say something about ds8k here

File: README.md [hunk 3]
```diff
@@ -216,31 +204,31 @@ spec:
...
+      - name: demo-volume
         persistentVolumeClaim:
-          claimName: pvc-demo
-
-      #nodeSelector:
-      #  kubernetes.io/hostname: NODESELECTOR
+          claimName: demo-pvc-file-system
 
+#      nodeSelector:
+#        kubernetes.io/hostname: NODESELECTOR
```
Comment (oriyarde) [inline README.md]: I prefer HOSTNAME (instead of NODESELECTOR)

File: USAGE-DETAILS.md [hunk 4]
```diff
@@ -4,130 +4,131 @@
...
     Driver:            block.csi.ibm.com
-    VolumeHandle:      A9000:6001738CFC9035EB0000000000D1F68F
+    VolumeHandle:      SVC:60050760718106998000000000000543
     ReadOnly:          false
-    VolumeAttributes:      array_address=<IP>
-                           pool_name=gold
-                           storage.kubernetes.io/csiProvisionerIdentity=1565550204603-8081-block.csi.ibm.com
-                           storage_type=A9000
-                           volume_name=demo1_pvc-a04bd32f-bd0f-11e9-a1f5-005056a45d5f
+    VolumeAttributes:      array_address=baremetal10-cluster.xiv.ibm.com
```
Comment (oriyarde) [inline USAGE-DETAILS.md]: I don't think we should put this on the internet

**Comment by oriyarde** [comment]
do the same here as in: https://github.com/IBM/ibm-block-csi-driver/pull/139#issuecomment-604540058

File: README.md (hunk 1 above)
Comment (rivkap) [reply README.md]: make this lower case "s"

**Comment by rivkap** [comment]
this doesn't seem to be the the full readme that I was working on with @27149chen . I had updated there the copyright date to 2020 a while ago.

**Comment by oriyarde** [comment]
> this doesn't seem to be the the full readme that I was working on with @27149chen . I had updated there the copyright date to 2020 a while ago.

I think we should just copy-paste here after you merge your work in the operator:
https://github.com/IBM/ibm-block-csi-operator/pull/55

File: README.md (hunk 1 above)
Comment (igorgonibm) [reply README.md:21]: this already fixed

Comment (oriyarde) [reply README.md:21]: the "S" case wasn't lowered as rivka suggested, it was removed:
https://github.com/IBM/ibm-block-csi-operator/pull/55/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21

**Comment by oriyarde** [comment]
> > this doesn't seem to be the the full readme that I was working on with @27149chen . I had updated there the copyright date to 2020 a while ago.
> 
> I think we should just copy-paste here after you merge your work in the operator:
> [IBM/ibm-block-csi-operator#55](https://github.com/IBM/ibm-block-csi-operator/pull/55)

@igorgonibm I didn't mean to copy-paste the main "operator" title.
anyway, can this branch be deleted?

File: README.md (hunk 2 above)
Comment (igorgonibm) [reply README.md:146]: what?

File: USAGE-DETAILS.md (hunk 4 above)
Comment (igorgonibm) [reply USAGE-DETAILS.md:108]: Changed to <IP> as it wasbefore

File: README.md (hunk 2 above)
Comment (oriyarde) [reply README.md:146]: something about maximum length / that it's being ignored?

Comment (igorgonibm) [reply README.md:146]: prefix length is limited but not ignored.
For each storage the limit is different.  I can't thin of any short comment that can explain that, This explanation is for user guide.

Comment (oriyarde) [reply README.md:146]: ok

Comment (igorgonibm) [reply README.md:146]: So approve this one please: https://github.com/IBM/ibm-block-csi-driver/pull/161
The missing changes are here.

**Pull request merged**